pages/k8s/install-manual.md

```diff
@@ -3,112 +3,209 @@ wrapper_template: "base_docs.html"
+each category!
+
 
+#### Networking
 
-The Juju documentation has more information on [adding credentials][credentials] and
-[configuring a controller][controller-config].
+---
 
 
-<!-- COMMENTED OUT UNTIL PAGE REFERRED TO IS ADDED
-<div class="p-notification--caution">
+<div class="CNI">
+ <div class="row">
+ <div class="col-2 ">
+   <span>Calico</span>
+ </div>
+  <div class="col-6 ">
+   <span>Calico provides out-of-the-box support for the
+   NetworkPolicy feature of Kubernetes, along with different modes of
+   network encapsulation. <a href="/kubernetes/docs/cni-calico"> Read more...</a></span>
+  </div>
+  <div class="col-4 ">
+    <span><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/calico-overlay.yaml" class="p-button--positive">Download calico-overlay.yaml</a></span>
+  </div>
+</div>
+<br>
+<div class="row">
+<div class="col-2">
+  <span>Canal</span>
+</div>
+ <div class="col-6">
+  <span>Shorthand for "Calico and Flannel", this combination brings in Calico's support for the NetworkPolicy feature of Kubernetes, while utilizing Flannel's UDP-based network traffic.<a href="/kubernetes/docs/cni-canal"> Read more...</a></span>
+ </div>
+ <div class="col-4">
+   <span class="u-vertically-center"><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/canal-overlay.yaml" class="p-button--positive">Download canal-overlay.yaml</a></span>
+ </div>
+</div>
+<br>
+<div class="row">
+<div class="col-2">
+  <span>Tigera Secure EE</span>
+</div>
+ <div class="col-6">
+  <span>Tigera Secure EE is a commercial version of
+  Calico with additional enterprise features. As well as deploying the software, you will need to configure
+  it with the relevant licence. <a href="/kubernetes/docs/tigera-secure-ee"> Read more...</a></span>
+ </div>
+ <div class="col-4">
+   <span class="u-vertically-center"><a href="https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/overlays/tigera-overlay.yaml" class="p-button--positive">Download tigera-overlay.yaml</a></span>
+ </div>
+</div>
+</div>
+<br>
+
+<div class="p-notification--positive">
   <p markdown="1" class="p-notification__response">
     <span class="p-notification__status">Note:</span>
-To manually install CDK locally using LXD, it is necessary to use a specific container profile before deploying.  See the <a href="/kubernetes/clouds-lxd">notes on LXD</a> before deployment.
+By default, Charmed Kubernetes uses <em>Flannel</em> for networking. You can read more about CNI support <a href="/kubernetes/docs/cni-overview"> here </a>.
   </p>
 </div>
--->
 
+#### Cloud integration
+
+---
+
+<div class="integration">
+ <div class="row">
+ <div class="col-2 ">
+   <span>AWS integrator</span>
+ </div>
+  <div class="col-6 ">
+   <span>Enables native support such as EBS storage and ELB load balancers. <a href="/kubernetes/docs/aws-integration"> Read more...</a></span>
```

Comment:
The "native" seems out of place. Maybe "AWS-native"? Or just, "Enables support for EBS storage and..."